### ISSUE TITLE
Add page name as class for wrapper

### ISSUE THREAD

**Issue by enricribas** [surveyjs/surveyjs#717]
### Are you requesting a feature, reporting a bug or ask a question?

Not sure, I guess a new feature

### What is the current behavior?

There is no way to target a specific page with CSS

### How would you reproduce the current behavior (if this is a bug)?

We have certain pages that need different css styling. If each page had the name of the page as a class name, a world of possibilities would be available.

Thanks

**Comment by tsv2013** [comment]
Each panel/page/question DOM element id can be changed as mentioned [here](https://github.com/surveyjs/surveyjs/issues/382#issuecomment-336035722).
And thus can be addressed via CSS selectors.

I think different surveys wll have different object (panel/page/question) names, set by user via SurveyJS Editor and it's hard to write some common css rules.

`afterRender` events set gives an opportunity to mark pages and other objects with specific styles.

**Comment by enricribas** [comment]
Isn't ```afterRender``` fired... well...after it's rendered?
How can I add a class or change its id if it's already rendered?
Sorry, I'm confused. I also don't understand why it's hard to add the pagename as a class or use that as the id, but ok.

**Comment by andrewtelnov** [comment]
@enricribas We have the following event for question, [survey.onUpdateQuestionCssClasses](https://surveyjs.io/Documentation/Library/?id=surveymodel#onUpdateQuestionCssClasses).
I may add the similar event for panel and page. Will it work for you?

Thank you,
Andrew

**Comment by tsv2013** [comment]
Yes, `afterRender` fires after DOM element has been rendered. Is case of adding CSS classes to any element no matters when it was done, the CSS rules will be applied immediately.

If you add a CSS class based on the page name, how you are going to create a corresponding css rule? You need to load a style sheet on the page, this style sheet sould mention all needed pre-defined CSS classes and your end-user sould give this names (selectors) to the pages of the survey. User should to know the right names and use them. It seems as rahter synthetical scenario. Probably I missed something. Could you please describe your real case?

**Comment by enricribas** [comment]
There is obviously some misunderstanding. :)

We have a multiple page survey. On one page, let's say a page named 'personal_info', we want the background to have an image or different colour. so CSS rule would be simple.

```
.surveyjs .personal_info {
   background-color: blue; // or whatever
}
```
For other pages, we don't want that colour. Another example is a survey question that is set up in two columns and everything is misaligned, so I need to fix it, but I don't want styles on that page to mess up other pages.

Our users do not create surveys, only we do. So we have full control of the page names used.

-----

I guess that magic happens with KO bindings. Well, supposed to happen, I can't get it to work. Updating ```options.page.cssClasses.root = 'something'``` isn't working for me. :( 
Maybe it has to do with React? No idea.

**Comment by enricribas** [comment]
To your other question, ``` survey.onUpdateQuestionCssClasses ``` does work but I *think* I can only add classes on the question level not the page level, unless there is a link to the page element somewhere I missed.

**Comment by enricribas** [comment]
Interestingly, adding ```survey.currentPage.cssClasses.root = ...``` does work but only on the next page, but not on the currentPage

**Comment by andrewtelnov** [comment]
@enricribas My suggestion is to ass survey.onUpdatePageCssClasses event. It doesn't exist, I may add it and it will work similar to survey.onUpdateQuestionCssClasses.
about survey.currentPage.cssClasses.root, it doesn't work because the first page is already rendered before you set the code.

Thank you,
Andrew
